src/internal/modules/charge-information/pre-handlers.js

```diff
@@ -32,6 +32,22 @@ const getFilteredChangeReasons = async type => {
   }
 };
 
+/**
+ * Removes incomplete charge elements to avoid breaking the UI
+ * and resets the session draftCharge info
+ * @param {*} request
+ */
+const loadValidatedDraftChargeInformation = async request => {
+  const { licenceId } = request.params;
+  const chargeVersionWorkFlowId = getChargeVersionWorkflowId(request);
+  const draftChargeInformation = await loadDraftChargeInformation(request);
+  // filter out incomplete charge elements when they have used the back button
+  draftChargeInformation.chargeElements = draftChargeInformation.chargeElements.filter(element => isEmpty(element.status));
```

Comment:
Is element.status normally an array or object? If not, is the isEmpty function correct here?

Reply:
Yeah it will be either exist or not as a property of charge element. So if status is present then it will discard that item in the array.